Makefile

```diff
@@ -14,6 +14,47 @@ ORDERED_PACKAGES:=fil_actors_runtime \
                   fil_actor_verifreg \
                   fil_builtin_actors_bundle
 
+# How much to "bump" the version by on release.
+BUMP ?= patch
+
+# Print out the current "bundle" version.
+version:
+	@cargo metadata -q --format-version=1 --no-deps | jq -r '.packages[] | select(.name == "fil_builtin_actors_bundle") | .version'
+
+# Run cargo check
+check:
+	cargo check --workspace --tests --benches --lib --bins --examples
+
+# Run cargo test (checking first)
+test: check
+	cargo test --workspace
+
+# Release a new version. Specify the version "bump" with BUMP
+release: check_clean check_deps test
+	echo "$(ORDERED_PACKAGES)" | xargs -n1 cargo set-version --bump $(BUMP) -p
+	cargo update --workspace
+	git commit -a -m "Release $$($(MAKE) --quiet version)"
```

Comment:
I personally don't like when things `git` for me, but I can see the utility here, I guess.

Reply:
Yeah, fair enough. Removed and documented.